packages/x-codemod/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-codemod",
-  "version": "6.0.0-alpha.9",
+  "version": "6.0.0-alpha.11",
```

Comment:
If we bump to the latest version (i.e. `alpha.11`) for other packages, should we do the same for `@mui/x-license-pro`?

Reply:
For the `x-license-pro` see https://github.com/mui/mui-x/pull/7144#discussion_r1043527877